tests/upload.js

```diff
@@ -373,4 +373,29 @@ describe("File upload", function () {
                 done();
             })
     });
+
+    it('With pre and post transformation', function (done) {
+        const fileOptions = {
+            fileName: "test_file_name",
+            file: "test_file_content",
+            transformation: { pre: 'w-100', post: [{type: 'transformation', value: ''}]}
```

Comment:
Don't allow empty `value`. It should result in error. Add test case.
`value` is optional but if provided, it can't be empty IMO.

Reply:
added checks for this and also added test suits to assert this